packages/expo/tsup.config.ts

```diff
@@ -18,6 +18,7 @@ export default defineConfig(overrideOptions => {
     minify: false,
     sourcemap: true,
     legacyOutput: true,
+    skipNodeModulesBundle: true,
```

Comment:
Why do we need to add this?

Reply:
Nice catch, this was a leftover from a test I was doing, removed!